cspell.json

```diff
@@ -250,6 +250,7 @@
     "subpkg",
     "supercede",
     "syml",
+    "syncer",
```

Comment:
"Syncer" is not really an English word, I guess. "Synchronizer" is the proper word.

Reply:
It is OK. Synchronizer is too long.